src/components/MovieDetails/MovieDetails.scss

```diff
@@ -1,13 +1,45 @@
-.movie-details-card {
-  width: 30em;
-  margin: 2em;
+.movie-details {
+  margin: 10em 3em 0em 3em;
+  section:first-child {
+    h1 {
+      font-size: 3em;
+    }
+  }
+  section:nth-child(2) {
+    margin-top: 1.5em;
+  }
+  section:nth-child(3) {
+    font-size: 1.5em;
+    margin-top: 3em;
+  }
 }
 
-.img-background {
-  width: 60em;
-  opacity: .25;
+.movie-details-card {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
 }
 
-.img-poster {
-  width: 20em;
+.movie-images {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1em;
+  margin-top: 7em;
+  img:first-child {
+    width: 20em;
+    margin: .5em 0em;
+  }
+  img:nth-child(2) {
```

Comment:
This is a really clever way of solving the problem of having a dynamically created background. Love the clear and reusable logic!